### PR TITLE
AV-157950 : Bundle changes for AKO Operator 1.8.1 Release and RH Cert…

### PR DESCRIPTION
**Pull request by DixitAakash**
This PR includes the required AKO Operator Bundle changes, for 1.8.1 Release and RH Certification.

### CODE REVIEW

**Comment by DixitAakash** [comment]
build ako

File: ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -78,24 +80,27 @@ metadata:
...
       ]
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.11.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.16.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     operatorframework.io/suggested-namespace: avi-system
-    containerImage: projects.registry.vmware.com/ako/ako-operator@sha256:378c4bb64cfff509193930b7bdf981c299e306c4bebf6f5f3197255c5c0e38dd
-  name: ako-operator.v1.6.3-stable
+    containerImage: projects.registry.vmware.com/ako/ako-operator@sha256:c556d055a9d6773c9a41655051f831dc83e548ee8d505a3794f1590fa6b8ce32
+  name: ako-operator.v1.8.1-stable
```
Comment (akshayhavile) [inline ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml:103]: @DixitAakash: name can be just `ako-operator.v1.8.1`

**Review by akshayhavile — COMMENTED** [review]
@DixitAakash : Are we still uses openshift version 4.6 in annotation.yaml? What is the use case of that version? Should we upgrade that?

File: ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (DixitAakash) [reply ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml:103]: This is the oldest version of Openshift, staring which AKO Operator will be supported. So, this annotation **com.redhat.openshift.versions: v4.6** means that AKO Operator will be uploaded to, and be available on Operator Hub of Openshift version v4.6 to v4.11 (latest version).

Comment (DixitAakash) [reply ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml]: name changed to **ako-operator.v1.8.1**

**Review by akshayhavile — APPROVED** [review]

**Comment by akshayhavile** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Pull request merged**